AmplicaLabs/social-app-template
# Always display full handle

# Purpose
The goal of this PR is to always display the full handle rather than displayHandle or base_handle.

Closes #78 

## Solution
- Update account service, backend, and frontend to handle this type change.

### Change summary
- Changed the types to always include the full handle object.
- Changed the account-service to always return the full handle object in https://github.com/AmplicaLabs/account-service/commit/afa4beff76cf131b63d15c17048e7b6f0c2a2214
- Updated the frontend to handle this object.


## Steps to Verify
1. Create 2+ accounts
2. See correct avatar
3. See the UI always display the url with the suffix


## Additional details / screenshot
- 
<img width="869" alt="Screenshot 2024-07-03 at 2 26 40 PM" src="https://github.com/AmplicaLabs/social-app-template/assets/43625033/a0460fbc-c5a8-4947-a5b4-1ae01b5bc437">
